Makefile

```diff
@@ -118,13 +118,15 @@ docs: crdoc manifests ## Generate docs.
 CLUSTER_PROVIDER ?= kind
 LOCAL_DEV_CLUSTER_VERSION ?= main
 GRAFANA_ENABLE ?= true
```

Comment:
Grafana enable will deploy prometheus by default

Reply:
+1 .. For this PR, it disables Monitoring in CI. 
I will fix the makefile issue  in a new pr #386